components/gcp/dataflow/launch_template_op.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+import re
+import time
+
+from googleapiclient import errors
+
+from kfp.gcp.core.base_op import BaseOp
+from kfp.gcp.core import name_utils
```

Comment:
I don't think you want to expose name_utils. You can do relative imports like https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/compiler/_component_builder.py#L25

Reply:
>I don't think you want to expose name_utils. You can do relative imports

AFAIK relative imports do not prevent exposure.